src/dosext/dpmi/memory.c

```diff
@@ -229,7 +229,7 @@ int dpmi_alloc_pool(void)
 {
     uint32_t memsize = dpmi_mem_size();
 
-    if (dpmi_base < dpmi_lin_rsv_base || dpmi_base + memsize >
+    if (dpmi_base < dpmi_lin_rsv_base || dpmi_base <
 	    dpmi_lin_rsv_base + dpmi_lin_mem_rsv()) {
```

Comment:
First part of `||` is now redundant.
Probably this check can completely be removed?